htmlSanityCheck-core/src/test/groovy/org/aim42/htmlsanitycheck/check/BrokenHttpLinksCheckerSpec.groovy

```diff
@@ -112,7 +112,7 @@ class BrokenHttpLinksCheckerSpec extends Specification {
     def "one syntactically correct http URL is ok"() {
         given: "an HTML page with a single correct anchor/link"
         String HTML = """$HtmlConst.HTML_HEAD 
-                <a href="http://${CustomHostNameResolver.WIREMOCK_HOST}:$port/google">google</a>
+                <a href="http://google.com:$port">google</a>
```

Comment:
See my remark on the previous comment, this could have been done in the other commit as well.

Reply:
Actually this was intentional from my side, as i wanted to change all the links in one commit

Reply:
Sorry, this is also OK!